docs/dictionary-and-concepts/dictionary-charges.md

```diff
@@ -10,32 +10,42 @@ The terms listed here are all terms relating to the topics of defining charges.
 |----------|-------------|
 | **A** |   |
 | **B** |   |
+| Business Reason Code | Indicates the business context, e.g. D10 = 'Request update charge information', which covers all three charge operation types; 'Create', 'Update' and 'Stop' |
```

Comment:
So, if the value is always the same then it doesn't add any value. Why don't we just remove it?

Reply:
Additional BusinessReasonCodes will follow, when we expand the domain with Charge Links and Requests (from other processes or direct request from Market Participants).

Reply:
Ok. But shouldn't we remove it from our charge commands and events where it doesn't add any value (only an opportunity to make an invalid message)?

Reply:
It is a valid question. I think we should have this talk at our next house keeping. (I've added it there).